spec/latest/core/error.jsonld

```diff
@@ -0,0 +1,11 @@
+{
+  "@context": {
+    "hydra": "http://www.w3.org/ns/hydra/core#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "type": "@type",
+    "title": "hydra:title",
+    "detail": "hydra:description",
+    "status": "hydra:errorCode",
```

Comment:
As stated above, I propose to keep this simple

```suggestion
    "status": "hydra:statusCode",
```

Reply:
Simple is not accurate in this case - let's first decide on the `hydra:Status` and `hydra:Error` issue

Reply:
Which issue is that?

Reply:
The one discussed above in https://github.com/HydraCG/Specifications/pull/218/files#r515617548, I assume, @tpluscode?

Reply:
Indeed